content/css/sel4.css

```diff
@@ -286,3 +286,61 @@ a.divLink {
   font-size:20px;
   font-weight: bold;
 }
+
+.news {
+  width: auto;
+  height:auto;
+  padding-top:5px;
+  padding-left:10px;
+  padding-right:5px;
+  margin-top:20px;
+  margin-left:60px;
+  margin-right:60px;
+  border-radius: 10px;
+  background-color: #e5e9ee; /* ligher than hover D6DBE4 */
```

Comment:
Have you tried "neutralLightest"?  (`#EBEEF5`)

Reply:
done. thanks